src/Symfony/Component/Serializer/Tests/Fixtures/StaticConstructorInstantiator.php

```diff
@@ -11,22 +11,24 @@
 
 namespace Symfony\Component\Serializer\Tests\Fixtures;
 
-use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+use Symfony\Component\Serializer\Instantiator\Instantiator;
 
 /**
- * @author Guilhem N. <egetick@gmail.com>
+ * @author Baptite Leduc <baptiste.leduc@gmail.com>
  */
-class StaticConstructorNormalizer extends ObjectNormalizer
+class StaticConstructorInstantiator extends Instantiator
 {
     /**
      * {@inheritdoc}
      */
-    protected function getConstructor(array &$data, string $class, array &$context, \ReflectionClass $reflectionClass, $allowedAttributes): ?\ReflectionMethod
+    protected function getConstructor(\ReflectionClass $reflectionClass): ?\ReflectionMethod
     {
+        $class = $reflectionClass->getName();
+
         if (is_a($class, StaticConstructorDummy::class, true)) {
```

Comment:
Should this be `\is_a(/* */)` ?

Reply:
`\` prefix are only used on OPCODE optimized methods, `is_a` is not one of them.

Reply:
Actually, if I understood it correctly, even if you don't have a specific optimized opcode it also gets rid of a function name runtime resolution, and deambiguates the call in case a function with the same name is declared in the namespace.

Reply:
Actually this is a cs-fixer that is not used in Symfony, so I don't think I should add it :wink:

Reply:
Yeah, I agree this is minor, this is fine anyway.